src/styles/ClubPage.module.css

```
@@ -17,18 +17,18 @@
 	border-radius: 10px;
 }
 
-table {
+.infoTable table {
 	margin: 0 auto;
 }
 
-table, th, td {
+.infoTable table, th, td {
 	border: 1px solid #ddd;
 }
 
-th, td {
+.infoTable th, td {
```

```suggestion
.infoTable table, .infoTable th, .infoTable td {
  border: 1px solid #ddd;
}

.infoTable th, .infoTable td {
```
make sure you include these too